README.md

```diff
@@ -5,3 +5,95 @@
 Jepsen is a correctness verifcation tools for distributed systems for their
 consistency under special scenarios. This project intended to apply Jepsen on
 Cloud Spanner databases.
+
+##Installation
+Make sure you have Java 11, Python 3 and `kubectl` installed.
+
+Download this repo.
+
+`git clone https://github.com/googleinterns/jepsen-on-spanner.git`
+
+You need to set up a kubernetes cluster and port the settings to `kuberctl` as well. Using Google
+Kubernetes Engine, execute:
+ 
+```
+gcloud container clusters create [cluster-name]
+gcloud container clusters get-credentials [cluster-name]
+```
+
+##Usage
+
+`python main.py [OPTION]`
+
+####Options
+
+`--project` specify the ID of the GCP project
+
+`--instance` specify the Cloud Spanner instance to run test on
+
+`--database` specify the name of the database to be created and tested on
+
+`--delete, -d` cleanup kubernetes jobs / Spanner instance
+
+`--redeploy, -r` redeploy the Jepsen-on-spanner executable to remote image
+
+`--worker, -w` specify number of workers
+
+`--benchmark, -b` specify the type of benchmark to run; supports `linearizability` and `bank`
+
+`--job, -j` if specified, will run until invalid history found or error occurs
+
+####Example
+
+To run a linearizability benchmark once on 8 workers:
+
+`python main.py -w 8 -b linearizability --project [projectID] --instance [instanceID] --database
+[databaseID]`
+
+To run a bank benchmark on 5 workers, redeploy the image on kubernetes, until failure:
+
+`python main.py -w 5 -r -j -b bank --project [projectID] --instance [instanceID] --database
+                                  [databaseID]`
+
+To cleanup a previous job with 3 workers:
+
+`python main.py -w 3 -d --project [projectID] --instance [instanceID] --database [databaseID]`
+
+## Design Overview
```

Comment:
I would move it to the place above the installation. People would like to read the context before starting installation.